src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -13,11 +13,28 @@ xk6-browser is currently being developed as a [k6 extension](/extensions). You h
 
 The quickest way to get started is to [download a release binary from GitHub](https://github.com/grafana/xk6-browser/releases).
 
+We suggest this option if you don't want to build your own binary with [xk6](https://github.com/grafana/xk6), which can be challenging in some cases.
+
 ### Build from source
 
-If you're more adventurous or want to get the latest changes of the xk6-browser extension, you can also build from source. 
+If you're more adventurous or want to get the latest changes of the xk6-browser extension, you can also build a binary from source.
+
+To build a binary with the extension, first, ensure you have installed [Go](https://golang.org/doc/install) and [Git](https://git-scm.com/). Then run the following commands in a terminal:
```

Comment:
Putting instructions in ordered lists highlights that this is a procedure with a set sequence.

Reply:
In general, I'd agree, but in this case I think it's better if we keep it consistent with the `InstallationInstructions` element.

Besides, it's a sequence of just 2 steps, and the first one is more of a pre-requisite we mention in passing. If we make it a standalone step, we might want to expand on how to confirm that Go and Git are installed properly, minimum versions, and things like that. Since this is rather off-topic for this document (and most of our installation instructions), I think it's fine if we leave this inline, as it was.

Reply:
I disagree, not all instructions have to be formatted as a list, particularly not 1 or 2 step ones like here. I find the Go and Git list in the "Before you start" section on the "Output Extensions" page overly verbose, for no practical gain, IMO.

But this is a style preference, and you've been working on the broader docs more than me, so I'll defer to your opinion.

Updating the `InstallationInstructions` element is out of scope for this PR, but I'll accept your suggestion here.